### ISSUE TITLE
Sqlite migrations limitations read confusing

### ISSUE THREAD

**Issue by ErikEJ** [dotnet/EntityFramework.Docs#3962]
https://docs.microsoft.com/hr-hr/ef/core/providers/sqlite/limitations#migrations-limitations 

The initial paragraph and some sections further down seem to ignore that all migrations operations are listed as supported in the summary table. 

Propose that the paragraphs are deleted. Happy to submit a PR based on the outcome of this issue.. (if needed 😁 )

**Comment by roji** [comment]
Yeah, you seem to be right - do we have any more migrations on Sqlite which aren't supported even via a table rebuild? @bricelam

**Comment by bricelam** [comment]
This could definitely use some clarification. The thing I still want to communicate is that these operations will still throw but only when you do them out-of-band for things that aren't part of your EF model.
